### ISSUE TITLE
How does one check the correctness of their implementation?

### ISSUE THREAD

**Issue by RylanSchaeffer** [pytorch/captum#795]
I want to use captum to try to understand a fine-tuned language model using Integrated Gradients. I have an implementation that runs without raising any errors. Before looking at the results, what are good checks to tell me that my implementation is correct?

I'm asking because I know deep learning models have a tendency to fail silently and in weird ways :grimacing: 

The usual approaches (e.g. plotting network inputs or outputs or loss curves) don't really make sense here.
